chia/plotting/check_plots.py

```diff
@@ -173,7 +174,9 @@ def process_plot(plot_path: Path, plot_info: PlotInfo, num_start: int, num_end:
                 # Some plot errors cause get_qualities_for_challenge to throw a RuntimeError
                 try:
                     quality_start_time = round(time() * 1000)
-                    for index, quality_str in enumerate(pr.get_qualities_for_challenge(challenge)):
+                    for index, quality_str in enumerate(
+                        pr.get_qualities_for_challenge(challenge, DEFAULT_CONSTANTS.PLOT_DIFFICULTY_INITIAL)
```

Comment:
for v2 plots we get the quality string from the parital proof with some calculation, its not obvious to me that get_qualities_for_challenge is relevant for v2, if we want it we need to make sure qualities and partial proofs are returned in the same order so we know which quality belongs to which proof

we would also probably just call the two methods instead of calculating quality string for each partial 

i think get_partial_proofs and something like get_qualties_from_partials is better